corehq/apps/reports/standard/cases/filters.py

```diff
@@ -149,6 +164,43 @@ def get_value(cls, request, domain):
         return ret
 
 
+class SensitiveCaseProperties(CaseListExplorerColumns):
+    slug = "sensitive_properties"
+    label = gettext_lazy("De-identify options")
+    template = "reports/filters/sensitive_columns.html"
+    EXCLUDE_PROPERTIES = [
```

Comment:
If this is only to be used in `get_column_suggestions()` you could move it into there.